packages/api/src/controllers/playback.ts

```diff
@@ -19,33 +22,56 @@ type PlaybackInfo = {
     playbackPolicy?: Asset["playbackPolicy"] | Stream["playbackPolicy"];
     source: {
       // the only supported format is HLS for now
-      hrn: "HLS (TS)";
-      type: "html5/application/vnd.apple.mpegurl";
+      hrn: "HLS (TS)" | "MP4";
+      type: "html5/application/vnd.apple.mpegurl" | "html5/video/mp4";
       url: string;
+      rendition?: string;
     }[];
   };
 };
 
-const newPlaybackInfo = (
+type AssetInfo = {
+  playbackUrl: string;
+  playbackPolicy?: Asset["playbackPolicy"];
+  files?: Asset["files"];
+  staticPlaybackUrl?: string;
+};
+
+function newPlaybackInfo(
   type: PlaybackInfo["type"],
   hlsUrl: string,
   playbackPolicy?: Asset["playbackPolicy"] | Stream["playbackPolicy"],
+  staticFilesPlaybackUrls?: string[],
   live?: PlaybackInfo["meta"]["live"]
-): PlaybackInfo => ({
-  type,
-  meta: {
-    live,
-    playbackPolicy,
-    source: [
-      {
-        hrn: "HLS (TS)",
-        type: "html5/application/vnd.apple.mpegurl",
-        url: hlsUrl,
-      },
-    ],
-  },
-});
+): PlaybackInfo {
+  let playbackInfo: PlaybackInfo = {
+    type,
+    meta: {
+      live,
+      playbackPolicy,
+      source: [
+        {
+          hrn: "HLS (TS)",
+          type: "html5/application/vnd.apple.mpegurl",
+          url: hlsUrl,
+        },
+      ],
+    },
+  };
+  if (staticFilesPlaybackUrls && staticFilesPlaybackUrls.length > 0) {
+    for (let staticUrl of staticFilesPlaybackUrls) {
+      const rendition = staticUrl.split("/").pop().replace(".mp4", "");
```

Comment:
Let's have this as an explicit field in the `files[]` object please! hahahah

Also, I think a resolution or file size would be better here. Idk we should have logic playing arbitrary rendition names like `720p60f` or sth like that 😆

Reply:
Ahahah yep. cc'ing @emranemran @mjh1 on why we need the probing in the output transcoded file. Currently I have just the size, I think the player could use that but wouldn't be able to know exactly which resolution it's using

Reply:
sure, i'll open a ticket for adding width and height fields. we can get this from the mediaconvert job status so wouldn't necessarily need to probe the output.

Reply:
Makes sense. 
@mjh1: filed this for you to just use mc job status for now to unblock studio team: https://github.com/livepeer/catalyst-api/issues/428

And i can take this one to add ffprobe logic to generate more metadata on output files: https://github.com/livepeer/catalyst-api/issues/429